src/System.Runtime.Caching/tests/System.Runtime.Caching/MemoryCacheTest.cs

```diff
@@ -1049,13 +1051,28 @@ public void TestCacheSliding()
                 mc.Add("expire5", "5", cip);
 
                 Assert.Equal(6, mc.GetCount());
-
+                // The loop below would sleep for ~5 seconds total (in 50 intervals).
```

Comment:
Should the test be outerloop then?

Reply:
Agreed - this particular one actually sleeps for ~9 seconds total. Perhaps it is possible to tighten it a bit but for now - made it outerloop.